save-demo/src/main/kotlin/com/saveourtool/save/demo/runners/cli/DiktatCliRunner.kt

```diff
@@ -0,0 +1,137 @@
+package com.saveourtool.save.demo.runners.cli
+
+import com.saveourtool.save.demo.diktat.DiktatDemoAdditionalParams
+import com.saveourtool.save.demo.diktat.DiktatDemoMode
+import com.saveourtool.save.demo.diktat.DiktatDemoResult
+import com.saveourtool.save.demo.diktat.DiktatDemoTool
+import com.saveourtool.save.demo.storage.ToolKey
+import com.saveourtool.save.demo.storage.ToolStorage
+import com.saveourtool.save.demo.utils.prependPath
+import com.saveourtool.save.utils.getLogger
+import com.saveourtool.save.utils.writeToFile
+import io.ktor.util.*
+import org.springframework.stereotype.Component
+import java.io.File
+import java.io.FileNotFoundException
+import java.nio.file.Path
+import kotlin.io.path.*
+
+/**
+ * Class that allows to run diktat/ktlint as command line application
+ *
+ * @property toolStorage
+ */
+@Component
+class DiktatCliRunner(
+    private val toolStorage: ToolStorage,
+) : CliRunner<DiktatDemoAdditionalParams, DiktatDemoResult> {
+    override fun getExecutable(workingDir: Path, params: DiktatDemoAdditionalParams): File {
+        val toolName = params.tool.name.lowercase()
+        val version = if (params.tool == DiktatDemoTool.DIKTAT) {
+            DIKTAT_VERSION
+        } else {
+            KTLINT_VERSION
+        }
+        val fileName = buildString {
+            append(toolName)
+            if (params.tool == DiktatDemoTool.KTLINT) {
+                append("-cli")
+            }
+            if (!osName().startsWith("Linux", ignoreCase = true) && !osName().startsWith("Mac OS", ignoreCase = true)) {
```

Comment:
```suggestion
            if (osName().startsWith("Windows", ignoreCase = true)) {
```
Since `.cmd` only makes sense on Windows anyway

Reply:
Yep, will do it